msprime/intervals.py

```diff
@@ -82,34 +159,70 @@ def size(self):
     @property
     def mean_rate(self):
         """
-        Return the weighted mean across all windows of the entire map.
+        The mean rate over this map, from map.start_position to map.end_position.
+        """
+        span = self.end_position - self.start_position
+        return self.total_mass / span
+
+    def at(self, x):
```

Comment:
I think it's clearer to call these ``get_rate`` and ``get_cumulative_mass``.

Reply:
We discussed this with Peter, and came up with "at" and "cumulative_at", ISTR?